mkosi/__init__.py

```diff
@@ -2221,7 +2299,13 @@ def check_inputs(config: Config) -> None:
             if not p.is_file():
                 die(f"Initrd {p} is not a file")
 
-    for script in config.prepare_scripts + config.build_scripts + config.postinst_scripts + config.finalize_scripts:
+    for script in (
+        config.sync_scripts +
+        config.prepare_scripts +
+        config.build_scripts +
+        config.postinst_scripts +
+        config.finalize_scripts
+    ):
```

Comment:
Possible alternative
```suggestion
    for script in itertools.chain(
        config.sync_scripts,
        config.prepare_scripts,
        config.build_scripts,
        config.postinst_scripts,
        config.finalize_scripts,
    ):
```